config/localization/en/app.json

```diff
@@ -1314,7 +1322,31 @@
       "TRADE_TO_WIN": "Trade to win",
       "TRADERS_JOINED": "Traders Joined",
       "USERS_REFERRED": "Users Referred",
-      "YOU_CAN_WIN_THESE": "You can win these in our {LEAGUES_LINK}"
+      "YOU_CAN_WIN_THESE": "You can win these in our {LEAGUES_LINK}",
+      "REWARD_HISTORY_TOOLTIP": "Time is displayed in Coordinated Universal Time (UTC)"
+    },
+    "TRADING_REWARDS" : {
+        "ABOUT": "About",
+        "EVENT": "Event",
+        "FOR_TRADING": "For trading {PERIOD}",
+        "FOR_V4":"{SUBJECT} for v4",
+        "GOVERNANCE": "Governance",
+        "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you need to export your secret phrase to Keplr Wallet.",
+        "LAUNCH_INCENTIVES":"Launch Incentives",
+        "LAUNCH_INCENTIVES_DESCRIPTION":"Each season, you can earn rewards for trading on dYdX Chain. Trade to earn your rightful spot on the leaderboard.",
```

Comment:
this is the most updated according to legal
<img width="375" alt="image" src="https://github.com/dydxprotocol/v4-localization/assets/9400120/36786730-a803-4ba7-ba28-855529d6f2c2">